internal/grpc/services/appprovider/appprovider.go

```diff
@@ -58,7 +58,8 @@ type config struct {
 	Demo      map[string]interface{} `mapstructure:"demo"`
 	IopSecret string                 `mapstructure:"iopsecret" docs:";The iopsecret used to connect to the wopiserver."`
 	WopiURL   string                 `mapstructure:"wopiurl" docs:";The wopiserver's URL."`
-	MimeTypes map[string]string      `mapstructure:"mimetypes"`
+	WopiBrURL string                 `mapstructure:"wopibridgeurl" docs:";The wopibridge's URL."`
+	MimeTypes map[string]string      `mapstructure:"mimetypes" docs:";List of supported mime types and corresponding file extensions."`
```

Comment:
Can you change this to `docs:"nil;List of supported...`? This isn't handled by cato, which results in the build to fail.

Reply:
Ah, that's why the build fail, I was indeed wondering (and did not have time to ask anyone earlier). Will fix it.